### PR TITLE
Test/schema

### PR DESCRIPTION
**Pull request by chitezh**

### CODE REVIEW

File: spec/models.spec/users.spec.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ describe('UserSchema', function() {
         expect(userSchemaObject['name'].options.required).toBe('Name must be present');
         expect(userSchemaObject['emailAddress'].options.required).toBe('Email address can not be empty');
     });
-    /*  it('should create a new user', function() {
+    it('should create a new user', function() {
         User.create(newuser, function(err, info){
```
Comment (hisabimbola) [inline spec/models.spec/users.spec.js]: replace info with user

File: spec/models.spec/users.spec.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ describe('UserSchema', function() {
...
             if(err){
                 expect(err).toBeDefined();
```
Comment (hisabimbola) [inline spec/models.spec/users.spec.js]: should err be defined?
err should be empty, test if it is empty

File: spec/models.spec/users.spec.js [hunk 3]
```diff
@@ -37,10 +37,42 @@ describe('UserSchema', function() {
         expect(info.name).toBe('Kingsley');
         expect(info.emailAddress).toBe('chitech247@gmial.com');
         });
-    });*/
+    });
      it('should throw an error when data is not sufficient, name or emailAddress', function() {
         User.create({name: 'Kingsley'}, function(err, info){
         expect(err).toBeDefined();
```
Comment (hisabimbola) [inline spec/models.spec/users.spec.js]: test info not to be defined

File: spec/models.spec/users.spec.js [hunk 4]
```diff
@@ -37,10 +37,42 @@ describe('UserSchema', function() {
...
         });
     });
+   it('should find users', function() {
+        User.find(function(err, info){
+            if(err){
```
Comment (hisabimbola) [inline spec/models.spec/users.spec.js]: no need for the "if" condition, you want to run the test always

File: spec/models.spec/users.spec.js [hunk 5]
```diff
@@ -37,10 +37,42 @@ describe('UserSchema', function() {
...
+                expect(err).toBeDefined();
+            }
+        var type = typeof info;
+        expect(type).toEqual('object');
+        expect(info.length).toBeGreaterThan(99);
```
Comment (hisabimbola) [inline spec/models.spec/users.spec.js]: this is not testing anything, can we assure that info.length will ever be less than 99?

you can rewrite it and test in another way

File: spec/models.spec/users.spec.js [hunk 6]
```diff
@@ -37,10 +37,42 @@ describe('UserSchema', function() {
...
+        });
+    });
+    it('should update users', function() {
+        User.find(function(err, info){
```
Comment (hisabimbola) [inline spec/models.spec/users.spec.js]: replace all info with the appropriate thing

File: spec/models.spec/users.spec.js (hunk 4 above)
Comment (bosunolanrewaju) [inline spec/models.spec/users.spec.js]: :+1: on that, no need to check for conditions in tests

Comment (chitezh) [inline spec/models.spec/users.spec.js]: Thanks

On Thu, May 7, 2015 at 8:23 AM, Olatunbosun Olanrewaju <
notifications@github.com> wrote:

> In spec/models.spec/users.spec.js
> https://github.com/andela-kochu/freelance/pull/1#discussion_r29829572:
> 
> > ```
> >   it('should throw an error when data is not sufficient, name or emailAddress', function() {
> >      User.create({name: 'Kingsley'}, function(err, info){
> >      expect(err).toBeDefined();
> >      });
> >  });
> > ```
> > -   it('should find users', function() {
> > -        User.find(function(err, info){
> > -            if(err){
> 
> [image: :+1:]
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/andela-kochu/freelance/pull/1/files#r29829572.

## 

_--_
_See what WIRED http://www.wired.com/2014/12/andela/ and CNN
http://www.cnn.com/2014/12/24/tech/africa-tech-startups-2014/ are saying
about Andela!_

_Kingsley Ochu | Software Developer | Andela_
(c) 234 (0) 80 649 49432 <08064949432> | Andela.co http://andela.co/ |
@kingsleyochu http://twitter.com/kingsleyochu

**Pull request merged**